### ISSUE TITLE
Attr responsetime_threshold doesn't work

### ISSUE THREAD

**Issue by fabioviana-hotmart** [russellcardullo/terraform-provider-pingdom#68]
When I try to use it I receive this error:

```
Error: Unsupported argument

  on main.tf line 14, in resource "pingdom_check" "this":
  14:     responsetime_threshold = 20

An argument named "responsetime_threshold" is not expected here.
```

I'm using the latest version (1.1.1)

**Comment by russellcardullo** [comment]
I've published a new version 1.1.2 which includes the fix for this in #36.

**Comment by fabioviana-hotmart** [comment]
Thank you.
